CHANGELOG.md

```diff
@@ -1,4 +1,30 @@
 ## 2.12.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.3.1486** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for ``enable_transit_firenet`` in **aviatrix_transit_gateway**
+2. Implemented support for AZURE provider in **aviatrix_firewall_instance** by adding ``username`` and ``password``
+3. Implemented support for AZURE provider in **aviatrix_vpc**
+4. New resources:
+  - **aviatrix_transit_firenet_policy**
+  - **aviatrix_firewall_management_access**
+  - **aviatrix_azure_spoke_native_peering**
+  - **aviatrix_azure_peer**
```

Comment:
1. Implemented support for Transit FireNet in Terraform
  - ``enable_transit_firenet`` in **aviatrix_transit_gateway**
  - ``username`` and ``password`` in **aviatrix_firewall_instance** for Azure (Transit) FireNet
2. New resources for Transit FireNet:
  - **aviatrix_transit_firenet_policy**
  - **aviatrix_firewall_management_access**
  - **aviatrix_azure_spoke_native_peering**
3. Implemented support for Azure VNet in **aviatrix_vpc** resource
  -  Supports regular and FireNet option
4. New resource **aviatrix_azure_peer** to replace **aviatrix_arm_peer**